scripts/tags.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# credit: https://github.com/cli/cli/blob/trunk/script/changelog
+
+function previous_tag() {
+  current_tag="${GITHUB_REF#refs/tags/}"
```

Comment:
This `GITHUB_REF` environment variable only exists within the GitHub Actions runtime. If we run the `changelog` or `release-changelog` make targets locally, the value of `$current_tag` will be `""` (empty string). This might mess up the prerelease conditional below but I'm unsure of the practical impact.

Reply:
Hmmm. This is a really good point. We could make this more robust by maybe using a fallback if `GITHUB_REF` is empty or something?

Reply:
Agreed, @mccurdyc I believe I mentioned this to you when reviewing the same change-set in go-fastly. We either just use HEAD or remove `current_tag` completely as it isn't really needed.

Reply:
Updated - https://github.com/fastly/cli/pull/122/commits/73aefbe028781aab34b468df97b9d9490a2d10b3#diff-97977000828f7cec0fad011d0491c836R7